docs/html/development/getting-started.rst

```diff
@@ -123,8 +123,22 @@ To build it locally, run:
 
 The built documentation can be found in the ``docs/build`` folder.
 
+What Next?
+===============
```

Comment:
The heading-underline should have the same number of characters as the heading.

```suggestion
==========
```

Reply:
Hi @pradyunsg,

I have pushed a new commit to do that. Once the PR is approved, I will squash both commits in one and merge the PR